easybuild/easyconfigs/p/Python/Python-2.7.17-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,293 @@
+        'checksums': ['c70410551488251b0fee67b460fb9a536af8d6f9f008ad10ac51f615b6a521b1'],
+    }),
+    ('configparser', '4.0.2', {
+        'checksums': ['c7d282687a5308319bf3d2e7706e575c635b0a470342641c93bea0ea3b5331df'],
+    }),
+    ('contextlib2', '0.6.0.post1', {
+        'checksums': ['01f490098c18b19d2bd5bb5dc445b2054d2fa97f09a4280ba2c5f3c394c8162e'],
+    }),
+    ('importlib_metadata', '1.5.0', {
+        'checksums': ['06f5b3a99029c7134207dd882428a66992a9de2bef7c2b699b5641f9886c3302'],
+    }),
+    ('pluggy', '0.13.1', {
+        'checksums': ['15b2acde666561e1298d71b523007ed7364de07029219b604cf808bfa1c765b0'],
+    }),
+    # more-itertools 5.0.0 is the most recent version still supporting Python 2.x
+    ('more-itertools', '5.0.0', {
+        'checksums': ['38a936c0a6d98a38bcc2d03fdaaedaba9f412879461dd2ceff8d37564d6522e4'],
+    }),
+    ('attrs', '19.3.0', {
+        'modulename': 'attr',
+        'checksums': ['f7b7ce16570fe9965acd6d30101a28f62fb4a7f9e926b3bbc9b61f8b04247e72'],
+    }),
+    ('wcwidth', '0.1.8', {
+        'checksums': ['f28b3e8a6483e5d49e7f8949ac1a78314e740333ae305b4ba5defd3e74fb37a8'],
+    }),
+    # pytest 4.6.x is the most recent version still supporting Python 2.x
+    ('pytest', '4.6.9', {
+        'checksums': ['19e8f75eac01dd3f211edd465b39efbcbdc8fc5f7866d7dd49fedb30d8adf339'],
+    }),
+    ('MarkupSafe', '1.1.1', {
+        'checksums': ['29872e92839765e546828bb7754a68c418d927cd064fd4708fab9fe9c8bb116b'],
+    }),
+    ('Jinja2', '2.11.1', {
+        'checksums': ['93187ffbc7808079673ef52771baa950426fd664d3aad1d0fa3e95644360e250'],
+    }),
+    ('packaging', '20.1', {
+        'checksums': ['e665345f9eef0c621aa0bf2f8d78cf6d21904eef16a93f020240b704a57f1334'],
+    }),
+    # sphinxcontrib-websupport 1.1.2 is the most recent version still supporting Python 2.x
+    ('sphinxcontrib-websupport', '1.1.2', {
+        'modulename': 'sphinxcontrib.websupport',
+        'checksums': ['1501befb0fdf1d1c29a800fdbf4ef5dc5369377300ddbdd16d2cd40e54c6eefc'],
+    }),
+    ('Pygments', '2.5.2', {
+        'checksums': ['98c8aa5a9f778fcd1026a17361ddaf7330d1b7c62ae97c3bb0ae73e0b9b6b0fe'],
+    }),
+    ('imagesize', '1.2.0', {
+        'checksums': ['b1f6b5a4eab1f73479a50fb79fcf729514a900c341d8503d62a62dbc4127a2b1'],
+    }),
+    ('docutils', '0.16', {
+        'checksums': ['c2de3a60e9e7d07be26b7f2b00ca0309c207e06c100f9cc2a94931fc75a478fc'],
+    }),
+    ('snowballstemmer', '2.0.0', {
+        'checksums': ['df3bac3df4c2c01363f3dd2cfa78cce2840a79b9f1c2d2de9ce8d31683992f52'],
+    }),
+    ('Babel', '2.8.0', {
+        'checksums': ['1aac2ae2d0d8ea368fa90906567f5c08463d98ade155c0c4bfedd6a0f7160e38'],
+    }),
+    ('alabaster', '0.7.12', {
+        'checksums': ['a661d72d58e6ea8a57f7a86e37d86716863ee5e92788398526d58b26a4e4dc02'],
+    }),
+    ('typing', '3.7.4.1', {
+        'checksums': ['91dfe6f3f706ee8cc32d38edbbf304e9b7583fb37108fef38229617f8b3eba23'],
+    }),
+    ('Sphinx', '1.8.5', {
```

Comment:
Add the same "latest for Py2" comment here

Reply:
done